scripts/distribution/ubuntu-packages/README.md

```diff
@@ -275,6 +275,25 @@ or only the most recent (e.g., last 10min)
 sudo journalctl -u concordium-node.service --since '10min ago'
 ```
 
+### Troubleshooting
+It can happen that the database of the concordium-node gets corrupted by a sudden shutdown of the concordium-node service e.g., by a sudden machine shutdown etc.
+
+This will result in an endless loop of service restarts as the concordium-node cannot startup successfully due to a database corruption.
+If this is the case then the log of the concordium-node will contain errors along the lines:
+
+```
+error, called at src/Concordium/GlobalState/Persistent/BlobStore.hs
+...
+concordium-node: warning: too many hs_exit()s
+concordium-node.service: Main process exited, code=exited, status=1/FAILURE
+concordium-node.service: Failed with result 'exit-code'.
+```
+
+In order to `recover` from such a situation, one may delete the contents (**except** the genesis.dat file) of the `data` folder.
+The default path for the `data` folder is `/var/lib/concordium/b6078154d6717e909ce0da4a45a25151b592824f31624b755900a74429e3073d/data/`.
```

Comment:
I think the hash should be replaced with, e.g., `$GENESIS_HASH` and explained that it might be different based on which environment it is.